benchmarks/linux_kernel/linux_testing_guide.md

```diff
@@ -0,0 +1,86 @@
+# Guide to running the VAST Linux kernel benchmark
+
+This document lists the steps to download, configure, and build the Linux
+kernel; and run `vast-front` on each of its compilation units.
+
+## Foreword
+
+These instructions were tested and written on an `XPS 13 9310` machine running
+`Ubuntu 22.04.4 LTS x86_64`.
```

Comment:
I would leave out the machine name, just mention the Ubuntu version, as that might be useful for reference/docker instances.